src/applications/accredited-representative-portal/routes.jsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import { Route, Routes } from 'react-router-dom-v5-compat';
-
 import App from './containers/App';
 import LandingPage from './containers/LandingPage';
 import POARequestsPage from './containers/POARequestsPage';
 import SignedInLayoutWrapper from './containers/SignedInLayoutWrapper';
+import POARequestDetails from './containers/POARequestDetails';
```

Comment:
We're suffixing containers associated with our routes with `Page`, so this should be `./containers/POARequestDetailsPage`.